playbooks/provider/lago/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+cluster: "openshift"
```

Comment:
Except for `inventory`, all the other variables are not being used by the Lago provider.
Why do we need this file?